### PR TITLE
HMAN-186

### PR DESCRIPTION
**Pull request by KleoG**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/HMAN-186

### Change description ###
Add logic for AWAITING_ACTUALS status for GET /hearing and GET /hearings. Add UTs.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/hmc/data/HearingEntity.java [hunk 1]
```diff
@@ -110,6 +111,32 @@ public Optional<HearingResponseEntity> getLatestHearingResponse() {
             : Optional.empty();
     }
 
+    public String getHearingStatus() {
```
Comment (danlysiak) [inline src/main/java/uk/gov/hmcts/reform/hmc/data/HearingEntity.java:114]: Just thinking, maybe we should name this a bit more explicitly to make it stand out more compared to the existing `getStatus()`?
```suggestion
    public String getDerivedHearingStatus() {
```

File: src/main/java/uk/gov/hmcts/reform/hmc/helper/GetHearingActualsResponseMapper.java [hunk 2]
```diff
@@ -37,7 +37,7 @@ public class GetHearingActualsResponseMapper extends GetHearingResponseCommonCod
 
     public HearingActualResponse toHearingActualResponse(HearingEntity hearingEntity) {
         val response = new HearingActualResponse();
-        response.setHmcStatus(getHearingStatus(hearingEntity));
```
Comment (danlysiak) [inline src/main/java/uk/gov/hmcts/reform/hmc/helper/GetHearingActualsResponseMapper.java:40]: We should be able to delete the private `getHearingStatus()` from within this class now too!

File: src/main/java/uk/gov/hmcts/reform/hmc/data/HearingEntity.java (hunk 1 above)
Comment (danlysiak) [inline src/main/java/uk/gov/hmcts/reform/hmc/data/HearingEntity.java:114]: Now it's in a common area as a `public` method, it'd be worth adding a couple of tests specifically for this method

Comment (KleoG) [reply src/main/java/uk/gov/hmcts/reform/hmc/data/HearingEntity.java]: Updated in: [c279ba1](https://github.com/hmcts/hmc-cft-hearing-service/pull/168/commits/c279ba1dcddc21323803f77eb741d161464b5de9)

**Pull request merged**